msteams-platform/toolkit/toolkit-v4/install-Teams-Toolkit-v4.md

```diff
@@ -0,0 +1,161 @@
+---
+title: Install Teams Toolkit v4
+author: zyxiaoyuer
+description: Learn about installation of Teams Toolkit v4 of different versions in Visual Studio Code, Visual Studio, and marketplace.
+ms.author: zhany
+ms.localizationpriority: medium
+ms.topic: overview
+ms.date: 07/29/2022
+zone_pivot_groups: teams-app-platform
+---
+
+# Install Teams Toolkit v4
+
+## Prerequisites
+
+::: zone pivot="visual-studio-code"
+
+Before installing Teams Toolkit for Visual Studio Code, you need to [download and install Visual Studio Code](https://code.visualstudio.com/Download).
+
+::: zone-end
+
+::: zone pivot="visual-studio"
+
+Before installing Teams Toolkit for Visual Studio, you need to [download and install Visual Studio 2022](https://aka.ms/VSDownload) using the Visual Studio Installer.
+
+::: zone-end
+
+::: zone pivot="visual-studio-code"
+
+## Install Teams Toolkit for Visual Studio Code
+
+You can install Teams Toolkit using **Extensions** in Visual Studio Code, or install it from the Visual Studio Code Marketplace.
+
+# [Visual Studio Code](#tab/vscode)
+
+1. Launch Visual Studio Code.
+1. Select **View > Extensions** or **Ctrl+Shift+X**. You can also open extensions by selecting the extensions :::image type="icon" source="images/vsc-ext-icon-v4.png" border="false":::icon from the Visual Studio Code activity bar.
+
+:::image type="content" source="images/install toolkit-1_2-v4.PNG" alt-text="Screenshot shows how to install.":::
+
+   The extensions marketplace pane appears.
+
+1. Enter **Teams Toolkit** in the search box.
+
+   :::image type="content" source="images/install-toolkit-2_2_1-v4.png" alt-text="Screenshot show the Toolkit.":::
+
+   Teams Toolkit appears in the search result list.
+
+1. Select **Teams Toolkit**, and then from the Teams Toolkit extension page that appears in the right pane, select  **Install**.
+
+:::image type="content" source="images/select-install-ttk_2-v4.png" alt-text="Screenshot shows install toolkit 4.0.0.":::
+
+   After successful installation of Teams Toolkit in Visual Studio Code, the Teams Toolkit icon appears in the Visual Studio Code activity bar.
+
+   :::image type="content" source="images/after-install_2-v4.PNG" alt-text="Screenshot shows after install view.":::
+
+# [Marketplace](#tab/marketplace)
```

Comment:
I'm not sure why we need to recommend both paths. It's more complicated to download the VSIX from the marketplace and manually install it as a VSIX. Unless there's a customer scenario I'm missing?